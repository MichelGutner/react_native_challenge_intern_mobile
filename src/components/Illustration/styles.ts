import styled from 'styled-components/native';

export const ImageView = styled.Image`
  width: 360px;
  height: 202px;
`;
