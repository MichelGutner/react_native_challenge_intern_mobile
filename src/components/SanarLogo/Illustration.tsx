import React, { Component } from 'react';
import { Image, StyleSheet } from 'react-native';

class Illustration extends Component {
  render() {
    return (
      <Image
        source={require('./LogosImage/illustration.png')}
        style={styles.image}
      />
    );
  }
}

export default Illustration;

const styles = StyleSheet.create({
    image: {
        width: 360, 
        height: 202
    }
})
