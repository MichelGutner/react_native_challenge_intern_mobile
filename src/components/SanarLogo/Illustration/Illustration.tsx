import React, { Component } from 'react';
import { ImageView } from './styles';

class Illustration extends Component {
  render() {
    return <ImageView source={require('./illustration.png')} />;
  }
}

export default Illustration;
