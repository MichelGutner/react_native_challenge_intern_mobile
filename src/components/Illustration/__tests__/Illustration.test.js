import React from 'react';
import renderer from 'react-test-renderer';
import Illustration from '../Illustration';
test('renders correctly', () => {
  const tree = renderer.create(<Illustration />).toJSON();
  expect(tree).toMatchSnapshot();
});
