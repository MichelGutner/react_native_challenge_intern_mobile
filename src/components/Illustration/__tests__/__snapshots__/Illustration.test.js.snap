// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Image
  source={
    Object {
      "testUri": "../../../src/components/Illustration/illustration.png",
    }
  }
  style={
    Array [
      Object {
        "height": 202,
        "width": 360,
      },
    ]
  }
/>
`;
